_posts/2019-01-08-data-driven-static-resume.adoc

```diff
@@ -0,0 +1,202 @@
+= Resumé-as-Code: Small Data Sourcing for Fun & Profit
+:page-layout: post
+:page-permalink: blog-small-data-driven-resume
+:page-date: 2019-01-09 17:22
+:page-comments: true
+:page-external-url:
+:page-categories: [small-data]
+:page-tags: [small data,Liquid,AsciiDoc,PDF,single sourcing,DRY]
+:page-author: Brian Dominick
+:toc: macro
+
+What better way to show off what I do as a software documentation platform developer (“DocOps engineer”) than by generating my own resumé with the technology stack I use professionally?
+Several prospective clients have remarked on the demonstrative novelty of this approach.
+If you also earn a crust from setting up documentation environments for engineering teams and the like, consider building your professional documentation from a single source to show off your skills.
+
+With its singular source and dual link:https://www.codewriting.org/brian-dominick-resume[HTML] and link:https://www.codewriting.org/assets/files/brian-dominick-resume.pdf[PDF] output, my resumé is a simple model implementation of the AJYL tech stack, consisting of *AsciiDoc*, *Jekyll*/*JAMstack*, *YAML*, and *Liquid*.
+
+In this post, I will walk you through how I set it all up, explaining how each component language and utility come into play.
+
+toc::[]
+
+== The Sources
+
+These two output versions (link:https://www.codewriting.org/brian-dominick-resume[HTML] | link:https://www.codewriting.org/assets/files/brian-dominick-resume.pdf[PDF]) are sourced from the exact same files, so they are always in sync;
+I do not have to remember to update two documents whenever I make a change.
+
+Most of the specific content items are stored as _small data_ objects arranged in a link:https://idratherbewriting.com/documentation-theme-jekyll/mydoc_yaml_tutorial.html[YAML] file, similarly to fields in a database.
```

Comment:
```suggestion
Most of the specific content items are stored as _small data_ objects arranged in a link:https://idratherbewriting.com/documentation-theme-jekyll/mydoc_yaml_tutorial.html[YAML] file, similar to fields in a database.
```

Reply:
Awesome catch. The _objects are similar to their counterparts in databases_ -- that is how most people will read this, no doubt. In my mind, it said they're _arranged similarly to their counterparts_.